packages/core/contracts/cross-chain-oracle/chain-adapters/test/Polygon_ChildMessengerMock.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "../Polygon_ChildMessenger.sol";
+
+contract Polygon_ChildMessengerMock is Polygon_ChildMessenger {
```

Comment:
kinda lame that you need extra mocks for polygon.

Reply:
That's only because there are internal methods to test